File: IotProxyServer/IotProxyClientLib/IotServer.cpp
#include "IotServer.h"

IotServer::IotServer(QObject *parent)
	:QObject(parent)
{
	conn=new IotServerConnection(this);
	mCommands=new IotServerCommands(conn,this);
}

IotServerConnection* IotServer::connection()
{
	return conn;
}

IotServerCommands* IotServer::commands()
{
	return mCommands;
}

File: WLIOTAndroidApplication/libwliotproxy/src/main/java/com/alterozoom/libwliotproxy/IotServer.java
package com.alterozoom.libwliotproxy;

public class IotServer
{
	private IotServerConnection srvConn;
	private IotServerCommands cmds;

	public IotServer()
	{
		srvConn=new IotServerConnection();
		cmds=new IotServerCommands(srvConn);
	}

	public IotServerConnection connection()
	{
		return srvConn;
	}

	public IotServerCommands commands()
	{
		return cmds;
	}
}
